alvr/server_openvr/cpp/alvr_server/alvr_server.cpp

```diff
@@ -374,12 +402,30 @@ void SetTracking(
         } else {
             if (g_driver_provider.left_controller && deviceMotions[i].deviceID == HAND_LEFT_ID) {
                 g_driver_provider.left_controller->onPoseUpdate(
-                    controllerPoseTimeOffsetS, deviceMotions[i], leftHand, controllersTracked
+                    controllerPoseTimeOffsetS,
+                    deviceMotions[i],
+                    leftHandSkeleton,
+                    controllersTracked && !useLeftHandTracker
+                );
+                g_driver_provider.left_hand_tracker->onPoseUpdate(
+                    controllerPoseTimeOffsetS,
+                    deviceMotions[i],
+                    leftHandSkeleton,
+                    controllersTracked && useLeftHandTracker
```

Comment:
Shouldn't the controllers and handtracking have different device motions?

Reply:
Well no, they are almost completely interchangeable, except one use full skeleton and one partial, and one is enable and the other is disabled. we could optimize things by not updating the skeleton for the disabled device

Reply:
Well no, they are almost completely interchangeable, except one use full skeleton and one partial, and one is enable and the other is disabled. we could optimize things by not updating the skeleton for the disabled device